editor/components/toolbar/index.js

```diff
@@ -9,27 +9,34 @@ import classNames from 'classnames';
 import './style.scss';
 import IconButton from 'components/icon-button';
 
-function Toolbar( { controls } ) {
+function Toolbar( { attributes, setAttributes, controls } ) {
```

Comment:
Well `attributes and `setAttributes` here don't mean "block attributes" and "update block attributes" but I see it as "toolbarState" "updateToolbarState" which is generic enough. Could be renamed but this would impact the `attributes` `setAttributes` used in the `isActive` and `onClick` of the block controls.